qml/components/chatInformationPage/ChatInformationPageContent.qml

```diff
@@ -0,0 +1,459 @@
+                headerText: qsTr("Chat Title", "group title header")
+                text: chatInformationPage.chatInformation.title
+
+                onSaveButtonClicked: {
+                    if(!editItem.errorHighlight) {
+                        tdLibWrapper.setChatTitle(chatInformationPage.chatInformation.id, textValue);
+                    } else {
+                        isEditing = true
+                    }
+                }
+
+                onTextEdited: {
+                    if(textValue.length > 0 && textValue.length < 129) {
+                        editItem.errorHighlight = false
+                        editItem.label = ""
+                        editItem.placeholderText = ""
+                    } else {
+                        editItem.label = qsTr("Enter 1-128 characters")
+                        editItem.placeholderText = editItem.label
+                        editItem.errorHighlight = true
+                    }
+                }
+            }
+            ChatInformationEditArea {
+                canEdit: (chatInformationPage.isPrivateChat && chatInformationPage.privateChatUserInformation.id === chatInformationPage.myUserId) || ((chatInformationPage.isBasicGroup || chatInformationPage.isSuperGroup) && chatInformationPage.groupInformation && (chatInformationPage.groupInformation.status.can_change_info || chatInformationPage.groupInformation.status["@type"] === "chatMemberStatusCreator"))
+                emptyPlaceholderText: qsTr("There is no information text available, yet.")
+                headerText: qsTr("Info", "group or user infotext header")
+                multiLine: true
+                text: (chatInformationPage.isPrivateChat ? chatInformationPage.chatPartnerFullInformation.bio : chatInformationPage.groupFullInformation.description) || ""
+                onSaveButtonClicked: {
+                    if(chatInformationPage.isPrivateChat) { // own bio
+                        tdLibWrapper.setBio(textValue);
+                    } else { // group info
+                        tdLibWrapper.setChatDescription(chatInformationPage.chatInformation.id, textValue);
+                    }
+                }
+            }
+
+            ChatInformationTextItem {
+                headerText: qsTr("Phone Number", "user phone number header")
+                text: (chatInformationPage.isPrivateChat && chatInformationPage.privateChatUserInformation.phone_number ? "+"+chatInformationPage.privateChatUserInformation.phone_number : "") || ""
+                isLinkedLabel: true
+            }
+
+            SectionHeader {
+                font.pixelSize: Theme.fontSizeExtraSmall
+                visible: !!inviteLinkItem.text
+                height: visible ? Theme.itemSizeExtraSmall : 0
+                text: qsTr("Invite Link", "header")
+                x: 0
```

Comment:
`x` is zero by default

Reply:
For SectionHeader, the default is `x: Theme.horizontalPageMargin` AFAICT

Reply:
Ah, you're getting rid of margins. OK.